lib/sql.js

```diff
@@ -1760,6 +1760,11 @@ SQLConnector.prototype.discoverExportedForeignKeys = function(table, options, cb
   this.execute(sql, cb);
 };
 
+SQLConnector.prototype.getDefaultSchema = function(options) {
+  throw new Error(g.f('{{getDefaultSchema}} must be implemented by' +
+  'the connector'));
+};
+
 SQLConnector.prototype.setDefaultOptions = function(options) {
```

Comment:
Maybe it's slightly better to have `getDefaultOptions` that returns an object of connector specific default options. I can live with the current version. But please add jsdocs.